src/Storages/StorageMySQL.cpp

```diff
@@ -250,8 +256,22 @@ void registerStorageMySQL(StorageFactory & factory)
         const String & password = engine_args[4]->as<ASTLiteral &>().value.safeGet<String>();
         size_t max_addresses = args.getContext()->getSettingsRef().glob_expansion_max_elements;
 
+        /// TODO: move some arguments from the arguments to the SETTINGS.
```

Comment:
Need to address that TODO.

Reply:
I don't want to mix this, I can submit it separately either on top of this PR or on top of upstream/master and rebase this PR on top of it.

Reply:
@azat ok lets do it in separate pr.